### PR TITLE
Mgmt: Fix live test error

### PR DESCRIPTION
**Pull request by ChenTanyi**
left: @weidongxu-microsoft com.azure.resourcemanager.samples.AppServiceSampleLiveOnlyTests.testManageLinuxWebAppWithTrafficManager

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/AppServiceSampleLiveOnlyTests.java [hunk 1]
```diff
@@ -136,8 +142,11 @@ public void testManageFunctionAppLogs() throws IOException {
         if (skipInPlayback()) {
             return;
         }
-        // require a longer read timeout in HttpClient
-        // e.g., new NettyAsyncHttpClientBuilder().readTimeout(Duration.ofMinutes(10))
+        azureResourceManager = AzureResourceManager.authenticate(
+            setReadTimeout(azureResourceManager.storageAccounts().manager().httpPipeline(), Duration.ofMinutes(10)),
+            new AzureProfile(azureResourceManager.tenantId(), azureResourceManager.subscriptionId(), AzureEnvironment.AZURE)
+        )
+            .withDefaultSubscription();
```
Comment (ChenTanyi) [inline sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/AppServiceSampleLiveOnlyTests.java:149]: would need to change `authenticate(httpPipeline, profile)` in https://github.com/Azure/azure-sdk-for-java/pull/16144

File: sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/kubernetescluster/samples/DeployImageFromContainerRegistryToKubernetes.java [hunk 2]
```diff
@@ -144,8 +145,9 @@ public static boolean runSample(AzureResourceManager azureResourceManager, Strin
                 .withServicePrincipalClientId(servicePrincipalClientId)
                 .withServicePrincipalSecret(servicePrincipalSecret)
                 .defineAgentPool("agentpool")
-                    .withVirtualMachineSize(ContainerServiceVMSizeTypes.STANDARD_D1_V2)
+                    .withVirtualMachineSize(ContainerServiceVMSizeTypes.STANDARD_D2_V2)
```
Comment (xccc-msft) [inline sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/kubernetescluster/samples/DeployImageFromContainerRegistryToKubernetes.java:148]: Why vm size changes here?

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appservice/samples/ManageLinuxWebAppCosmosDbByMsi.java [hunk 3]
```diff
@@ -167,13 +167,18 @@ public static boolean runSample(AzureResourceManager azure, String clientId) thr
             DockerClient dockerClient = DockerUtils.createDockerClient(azure, rgName, region,
                     azureRegistry.loginServerUrl(), acrCredentials.username(), acrCredentials.accessKeys().get(AccessKeyType.PRIMARY));
 
+            String imageName = "tomcat:7.0-slim";
             String privateRepoUrl = azureRegistry.loginServerUrl() + "/todoapp";
-            dockerClient.buildImageCmd(new File(ManageLinuxWebAppCosmosDbByMsi.class.getResource("/todoapp-cosmosdb/Dockerfile").getFile()))
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appservice/samples/ManageLinuxWebAppCosmosDbByMsi.java:171]: Long term, we might probably try fix this docker file (or rather, use JAR deployment directly).

File: sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/kubernetescluster/samples/DeployImageFromContainerRegistryToKubernetes.java (hunk 2 above)
Comment (ChenTanyi) [reply sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/kubernetescluster/samples/DeployImageFromContainerRegistryToKubernetes.java:148]: The `SYSTEM` agent pool doesn't support 1 cpu VM

File: sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appservice/samples/ManageLinuxWebAppCosmosDbByMsi.java (hunk 3 above)
Comment (ChenTanyi) [reply sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appservice/samples/ManageLinuxWebAppCosmosDbByMsi.java:171]: I think maybe JAR is better, corresponding to windows version.

**Review by xccc-msft — APPROVED** [review]

**Pull request merged**